src/radical/pilot/agent/scheduler/tagging.py

```diff
@@ -0,0 +1,312 @@
+
+__copyright__ = "Copyright 2013-2016, http://radical.rutgers.edu"
+__license__   = "MIT"
+
+
+from .continuous import Continuous
+
+
+# ------------------------------------------------------------------------------
+#
+# This is a simle extension of the Continuous scheduler which makes RP behave
+# like a FiFo: it will only really attempt to schedule units if all older units
+# (units with lower unit ID) have been scheduled.  As such, it relies on unit
+# IDs to be of the format `unit.%d'`.
```

Comment:
This comment does not match the class.

Reply:
Yup, agreed. I created a separate scheduler/file to test just the tagging part. Both the DA and the tagging work now. So combining them into the continuous scheduler where the tagging will be optional.

The tagging.py file won't exist, this was just temporary.